cartesian_velocity_controller:
    type: hiro_panda_moveit/CartesianVelocityController
    arm_id: panda
